app/assets/stylesheets/common/_markus.scss

```diff
@@ -159,16 +155,11 @@ textarea {
 }
 
 textarea {
-  // height: 2.5em;
   resize: none;
-
-  // &:focus {
-  //   height: 7.5rem;
-  // }
 }
 
 select {
-  @include border-radius($radius);
+  border-radius: $radius;

```

Comment:
Properties should be ordered background, border, border-radius, font-family, margin, padding-left